sdcm/sct_events/database.py

```diff
@@ -66,7 +64,7 @@ class DatabaseLogEvent(LogEvent, abstract=True):
 
 # pylint: disable=too-few-public-methods
 class ReactorStalledMixin(Generic[T_log_event]):
-    tolerable_reactor_stall: int = TOLERABLE_REACTOR_STALL
+    tolerable_reactor_stall: int = os.environ.get('SCT_REACTOR_STALL_TOLERABLE_THRESHOLD')
```

Comment:
Does it work? i think it wouldn't, because we didn't export config parameters to os.env. 
i think it is better to get it from node.parent_cluster.params["reacto_stall_tolerable_threshold"]

Reply:
@aleksbykov , running it in here with very small threshold and detector:
https://jenkins.scylladb.com/job/scylla-staging/job/fabio/job/PR5470_control_reactor_stall_tolerable/4/